.github/workflows/test.yml

```diff
@@ -5,6 +5,9 @@ on:
       - 'dependabot/*'
   schedule:
     - cron: '22 22 * * 5'
+env:
```

Comment:
Actually there is no need to use expressions here. The SHA and the repository name are available as environment variables per default: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
That’s true and also it’s not used. 😂